packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -223,12 +222,12 @@ class SkiaGoldClient {
       '--changelist', pullRequest,
       '--cis', 'cirrus',
       '--jobid', cirrusTaskID,
-      '--patchset', nthCommit,
+      '--patchset', commitHash,
```

Comment:
This will make ingestion sad, I believe. However, I can update gold to take the PatchSet ID (i.e. the commit hash) and turn it into the nth commit on our side.  This will also likely require a push to goldctl and a slightly different flag (--patchset_id)

Reply:
https://skia-review.googlesource.com/c/buildbot/+/256018 will have that (plus a CL to roll goldctl in CIPD)

Reply:
Ah I see. Thank you for adding that option - and so quickly! :+1:

Reply:
CIPD has the new version and ingestion servers should be able to take it